tls/s2n_client_hello.c

```diff
@@ -444,9 +444,14 @@ int s2n_parse_client_hello(struct s2n_connection *conn)
      * P-256 is our preferred fallback option because the TLS1.3 RFC requires
      * all implementations to support it:
      *
-     *     https://tools.ietf.org/rfc/rfc8446#section-9.1
-     *     A TLS-compliant application MUST support key exchange with secp256r1 (NIST P-256)
-     *     and SHOULD support key exchange with X25519 [RFC7748]
+     * https://tools.ietf.org/rfc/rfc8446#section-9.1
+     * A TLS-compliant application MUST support key exchange with secp256r1 (NIST P-256)
+     * and SHOULD support key exchange with X25519 [RFC7748]
+     *
```

Comment:
Can you keep the original formatting? This isn't a compliance comment, and I think the old indent made that clear.